pulsar-broker/src/main/java/org/apache/pulsar/broker/web/WebService.java

```diff
@@ -399,6 +399,7 @@ public void start() throws PulsarServerException {
             }
 
             server.setHandler(stats);
+            server.setStopTimeout(5000); // 5 seconds
```

Comment:
This should be removed from this PR as mentioned in https://github.com/apache/pulsar/pull/24198#issuecomment-2826737944